### ISSUE TITLE
Nano bot modifiers being disabled regularly (45 seconds)

### ISSUE THREAD

**Issue by ftrocchi** [ValkyrieofNight/EnvironmentalTech#140]
We keep falling out of the sky!  LOL.  This problem is most notable with the creative flight modifier, but it is doing the same thing for all modifiers.  Every 45 seconds it seems the effects get disabled.  Sometimes it re-enables, but sometimes it doesn't.  When it doesn't we have to go and break the nano bot beacon and replace it.

Here is a log snippet, and you can clearly see it is happening every 45 seconds...

```
[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:resistance CandyVette

[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:strength CandyVette

[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:regeneration CandyVette

[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:water_breathing CandyVette

[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:saturation CandyVette

[10:16:48] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:fire_resistance CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:resistance CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:strength CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:regeneration CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:water_breathing CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:saturation CandyVette

[10:17:33] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:fire_resistance CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:resistance CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:strength CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:regeneration CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:water_breathing CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:saturation CandyVette

[10:18:18] [INFO] [com.valkyrieofnight.et.m_multiblocks.m_nanobot.tile.NBBContBase:disableEffects:52]: Disable: minecraft:fire_resistance CandyVette
```

**Comment by ftrocchi** [comment]
That's what we though and looked at.  It is getting enough power for sure, and we have two different chunk loaders working on the chunk.  What is weird is users get the same disabled scenario at the same time.  So for instance we are all flying and then all of us fall out of the sky at the same time.

I'll check later tonight when get out of work and let you know.  Thanks for the quick reply!

**Comment by ValkyrieofNight** [comment]
Hmm that is strange I will look into that later tonight when I get home from work. What pack are you using and also what chunk loaders so I can try replicate the bug.

**Comment by ValkyrieofNight** [comment]
What version of ET are you using?

**Comment by thebigcarp321** [comment]
I just had this same thing happen to us. We just built the Ranged Nano Beacon. It seems if we are sitting still, the counter resets, but as soon as we move, the counter ticks all the way down to 0 and then starts all over again with the buffs being applied a few more seconds later. I added the health boost modifier and it damages us to where it removes all the extra hearts and then starts to reapply them all over again.

Unit is chunkloaded and is being provided power.

MC version 1.12.2
The Final Frontier version 4.0
ET version 1.12.2-2.0.13a
